dashboard/lib/pd/foorm/helper.rb

```diff
@@ -24,12 +24,35 @@ def get_survey_key(ws_submission)
     def get_index_for_survey_key(survey_key)
```

Comment:
I guess this whole method seems a little suspect for the long term, how are you thinking about it? Works in the short term and we figure something else out later?

Reply:
yeah I'm open to suggestions on making it better. Basically we need it to sort the surveys by (1) Pre surveys (sorted by module if applicable) (2) daily surveys (sorted by day) (3) Post surveys (sorted by module if applicable). (This sorts the tabs for the results view).

Reply:
Got it, seems like it'll work in the short term.

Reply:
cleaned this method up and added unit tests for this file